Strategies/ITVision.md

```diff
@@ -0,0 +1,168 @@
+# ESDC's IT Vision
+
+[HDLC](HumanDevelopmentLifeCycle.md) -&gt; [KPIs](IITB_KPIs.md) -&gt; IT Vision -&gt; Guiding Principles -&gt; IT Strategies (roadmap)
+
+The following is ESDC's IT Vision.
+It paints a vision of the future of IT within ESDC.
+It is **not** meant to be time bound, but rather as an ideal state to which ESDC IT needs to strive to get to.
+Such state will continuously evolve when new information and knowledge is captured, re-adjusting the vision in the process.
+
+The IT Vision complements the [HDLC](HumanDevelopmentLifeCycle.md) story which is taken from the point of view of the citizen, and the [KPIs](IITB_KPIs.md) which define the performance metrics needed to sustain the HDLC, by providing the point of view of the organization's IT capabilities.
+
+The IT Vision leverages the work done in the [IITB Way Forward](http://dialogue/grp/IITB-DGIIT-Gov-New-Nouveau/Documents/Departmental%20IMIT%20Plan/19-20%20Plans/IITB%20Moving%20Forward%20v2.docx), ensures alignment with [IITB's high level priorities](http://esdc.prv/en/iitb/corporate/Who_We_Are/Plans_and_Priorities/index.shtml), and brings the [GC's Digital Operations Strategic Plan 2018-2022](https://www.canada.ca/en/government/system/digital-government/digital-operations-strategic-plan-2018-2022.html) into ESDC's operational context.
+
+The ESDC IT Vision is explained using the [GC's Digital Operations Strategic Plan 2018-2022](https://www.canada.ca/en/government/system/digital-government/digital-operations-strategic-plan-2018-2022.html) six chapters:
+
+1. [User-centred, service orientation](#user-centred-service-orientation)
+2. [An open, collaborative, and accessible government](#an-open-collaborative-and-accessible-government)
+3. [Digital-first and digitally enabled by design](#digital-first-and-digitally-enabled-by-design)
+4. [Modern tech, technology practices and information stewardship](#modern-tech-technology-practices-and-information-stewardship)
+5. [A digitally enabled public service](#a-digitally-enabled-public-service)
+6. [Good digital governance](#good-digital-governance)
+
+## 1. User-centred, service orientation
+
+Human Interfaces that citizens interact with (e.g. websites, web applications, and mobile applications) are rarely provided by ESDC.
+Instead, **ESDC does not compete with the private sector** and seeks to leverage it to drastically improve citizen's user experience as well as [promoting Canadian innovation and skills](https://www.theglobeandmail.com/business/commentary/article-the-essentials-for-canadas-digital-economy/).
```

Comment:
I think it would be better to reference a TBS policy rather than a globe and mail article 

Ex: [Digital Operations Strategic Plan: 2018-2022](https://www.canada.ca/en/government/system/digital-government/digital-operations-strategic-plan-2018-2022.html)

Which reads ... 

- identifying partnerships and working with peers in other jurisdictions (provincial, territorial and municipal) with academia, with non-profits and with the private sector should be the norm
- being open by default allows public service research and evidence to be peer-reviewed by Canadians and invites collaboration
-  In October 2018, Canada began its term as lead government chair of the Open Government Partnership, a multilateral initiative comprising over 75 member nations that aims to secure concrete commitments to promote transparency, empower citizens, fight corruption, and harness new technologies to strengthen governance, in partnership with civil society and the **private sector**.

etc...

Reply:
ESDC does not compete with the private sector ==> ESDC **must not** compete with the private sector

Does not is present tense ... and presently we do compete (even if accidentally)